src/components/dashboard/PotentialBarriers.vue

```diff
@@ -74,8 +80,10 @@ const rows = [
 </script>
 
 <style scoped>
-.centered {
-  margin-left: auto;
-  margin-right: auto;
+.potential-barriers-container {
+  display: grid;
+  gap: 1.5rem;
+  grid-template-columns: repeat(auto-fit, minmax(min(15rem, 100%), 1fr));
```

Comment:
is the same styling as the gap chart card? if so, can we centralize/unify?

Reply:
Yes it is! I've centralized it into the `main.scss` as `signal-grid-container`